src/compositemark/boxplot.ts

```diff
@@ -171,6 +173,44 @@ export function normalizeBoxPlot(spec: GenericUnitSpec<Encoding<string>, BoxPlot
       }
     ]
   };
+
+  if (isMinMax) {
+    return result;
+  }
+
+  // tukey plot
+  return {
+    ...outerSpec,
+    layer: [
+      { // boxplot
+        transform,
+        layer: result.layer
+      }, {
+        transform: [
+          {
+            window: transform[0].aggregate.slice(0, 2),
+            frame: [null, null],
+            groupby: transform[0].groupby
+          }, {
+            filter: `(datum.${continuousAxisChannelDef.field} < datum.lower_box_${continuousAxisChannelDef.field} - ${extent} * (datum.upper_box_${continuousAxisChannelDef.field} - datum.lower_box_${continuousAxisChannelDef.field})) || (datum.${continuousAxisChannelDef.field} > datum.upper_box_${continuousAxisChannelDef.field} + ${extent} * (datum.upper_box_${continuousAxisChannelDef.field} - datum.lower_box_${continuousAxisChannelDef.field}))`
+          }
+        ],
+        mark: {
+          type: 'point',
+          ...(sizeValue ? {size: sizeValue} : {}),
+          ...getMarkDefMixins<BoxPlotPartsMixins>(markDef, 'outliers', config.boxplot)
+        },
+        encoding: {
+          [continuousAxis]: {
+            field: continuousAxisChannelDef.field,
+            type: continuousAxisChannelDef.type
+          },
+          ...encodingWithoutSizeColorAndContinuousAxis,
+          ...(size ? {size} : {})
```

Comment:
Why applying size here? Size of bar = width, size of point = area. Applying the same number would only lead to problematic result.